src/main/java/ome/logic/AdminImpl.java

```diff
@@ -1219,7 +1219,7 @@ private void sendEmail(Experimenter e, String newPassword) {
                             + "to user:"
                             + e.getOmeName()
                             + ". Please turn on the debug "
-                            + "mode in omero.properties by the: omero.mail.debug=true");
```

Comment:
This could probably be generalized, since really the user should use `bin/omero config`:

```
Please turn on debugging by setting omero.mail.debug=true
```